src/service.rs

```diff
@@ -1256,6 +1270,30 @@ impl ServiceStatus {
             exit_code: ServiceExitCode::from(&raw),
             checkpoint: raw.dwCheckPoint,
             wait_hint: Duration::from_millis(raw.dwWaitHint as u64),
+            pid: None,
```

Comment:
We tend to maintain the snakecase variants of the corresponding field names used in WinAPI, so it's easier to relate to the original WinAPI structures. The only exception is `exit_code` which is a combo of both `dwWin32ExitCode` and `dwServiceSpecificExitCode`.

So `SERVICE_STATUS_PROCESS .dwProcessId` would translate to something like `ServiceStatus.process_id`.

I am still looking at your PR but that's something that draw attention at first sight.

Reply:
Sorry I managed to miss this notification. You are completely correct on the naming consistency, I'll fix that up.